STUDENT_COURSE _MANAGEMENT/src/main/java/com/springboot/demo/studentcourse/repository/CourseRepository.java

```diff
@@ -0,0 +1,9 @@
+package com.springboot.demo.studentcourse.repository;
+
+import com.springboot.demo.studentcourse.enity.Course;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+
+public interface CourseRepository extends JpaRepository<Course,Integer> {
```

Comment:
can you try using crudrepository instead of jparepository

Reply:
changed to crud repository